manifests/server/puppetserver.pp

```diff
@@ -63,6 +66,7 @@
   $server_ruby_load_paths      = $::puppet::server::ruby_load_paths,
   $server_cipher_suites        = $::puppet::server::cipher_suites,
   $server_max_active_instances = $::puppet::server::max_active_instances,
+  $server_max_requests_per_instance = $::puppet::server::max_active_instances,
```

Comment:
If I could be a pain and ask you to realign the variables both here and the other files, it'd help when somebody edits it for something else later and feels the need to realign them!